common/src/java/com/zimbra/common/soap/AccountConstants.java

```diff
@@ -644,4 +644,11 @@ public class AccountConstants {
     public static final String E_SKIN_SELECTION_COLOR = "zimbraSkinSelectionColor";
     public static final String E_SKIN_FAVICON = "zimbraSkinFavicon";
     public static final String E_HOSTNAME = "hostname";
+
+    // MailRecall Uses
+    public static final String E_MAIL_RECALL_REQUEST = "MailRecallRequest";
```

Comment:
I don't see the handler registered for this soap request, is it in some extension?